src/core/lib/event_engine/windows/windows_engine.cc

```diff
@@ -496,8 +499,8 @@ EventEngine::ConnectionHandle WindowsEventEngine::Connect(
   bool success =
       ConnectEx(connection_state->socket()->raw_socket(), address.address(),
                 address.size(), nullptr, 0, nullptr, info->overlapped());
-  // It wouldn't be unusual to get a success immediately. But we'll still get an
-  // IOCP notification, so let's ignore it.
+  // It wouldn't be unusual to get a success immediately. But we'll still get
```

Comment:
Are you setting your word wrap to < 80 characters? The `.` falls on column 80. This should be reverted.

Reply:
reverted